test/sqllogictest/comparison.slt

```diff
@@ -28,3 +28,47 @@ query B
 select null isnull
 ----
 true
+
```

Comment:
Let's add a test case that shows that we're handling the difference between `is null` and `is unknown`